src/hubverse_transform/hub_config.py

```diff
@@ -65,3 +108,31 @@ def _get_tasks_config(self, tasks_file: str):
 
         with tasks_path.open() as f:
             return json.loads(f.read())
+
+    def _get_task_id_values(self, task: dict) -> dict[str, set]:
+        """Return a dict of ids and values for a specific modeling task."""
+        task_id_values = dict()
+
+        # create a dictionary of all tasks ids and values for this task
+        task_ids = {task_id[0]: task_id[1] for task_id in task.get("task_ids", {}).items()}
+
+        # flatten the dictionary values for each task_id (i.e., combine "required" and "optional" lists)
+        for task_id in task_ids.items():
+            task_id_values[task_id[0]] = set(
+                itertools.chain.from_iterable([value or [] for value in task_id[1].values()])
+            )
+
+        return task_id_values
+
+    def _get_data_type(self, value: int | bool | str | date | float) -> type:
```

Comment:
Got ahead of myself here: this is the helper that will eventually determine the data type of our task_ids and output_type_ids, but it's not being used yet.